client/src/app/gateways/export/csv-export.service/csv-export-for-backend.service.spec.ts

```diff
@@ -114,7 +114,7 @@ describe(`CsvExportForBackendService`, () => {
 
     it(`test dummy export method with default settings`, () => {
         service.dummyCSVExport(
-            { proper: `Fancy`, tea: `Tea`, strength: `Muscle` },
+            Object.keys({ proper: `Fancy`, tea: `Tea`, strength: `Muscle` }),
```

Comment:
Just use the list of keys at this point, no use keeping this as an object

Reply:
done.